metricbeat/module/http/server/server.go

```diff
@@ -19,9 +19,15 @@ package server
 
 import (
 	"fmt"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/elastic/beats/v7/libbeat/common"
+	"github.com/elastic/beats/v7/libbeat/logp"
+	"github.com/elastic/beats/v7/metricbeat/helper/server"
 
```

Comment:
```suggestion
```

Reply:
will not be an issue with `NewHttpServerWithHandler`